Ryujinx.HLE/HOS/Services/Am/AppletAE/AllSystemAppletProxiesService/SystemAppletProxy/ISelfController.cs

```diff
@@ -298,6 +298,17 @@ public ResultCode ReportUserIsActive(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [CommandHipc(67)] //3.0.0+
+        // IsIlluminanceAvailable() -> bool
+        public ResultCode IsIlluminanceAvailable(ServiceCtx context)
+        {
+            // NOTE: This should call through to Lbl, but there's no situation where we'd want false.
+            context.ResponseData.Write(true);
+            
+            Logger.Stub?.PrintStub(LogClass.ServiceAm);
+            return ResultCode.Success;
```

Comment:
```suggestion

            return ResultCode.Success;
```
One empty line before a return